### PR TITLE
return json parse result

### PR DESCRIPTION
**Pull request by mehtaphysical**
Should return the json result.

### CODE REVIEW

**Comment by Karunamon** [comment]
Hi there, and thanks for the contribution!

I haven't had a chance to actually test this in the wild yet, but are some of these changes necessary to support new versions of Concourse?

File: scripts/bitbucket.py [hunk 1]
```diff
@@ -91,11 +91,11 @@ def post_result(url, user, password, verify, data, debug):
         err("Commit: " + str(commit_sha))
 
     key = commit_sha
-    if j['params']['key']:
+    if j['params'].get('key', False):
```
Comment (Karunamon) [inline scripts/bitbucket.py:94]: This is the only change that sticks out to me - I'm trying to read my mind 4 months in the past, but I think the idea here is that if the key doesn't exist (means we got JSON that didn't have it for some reason), the job will bomb right here with a KeyError and give the user appropriate feedback to look at their Concourse config.

**Comment by aecepoglu** [comment]
Can we just merge the 1st 2 commits of this PR already? They're critical to get this resource working.

**Comment by Karunamon** [comment]
@aecepoglu, Can you confirm that the resource as existing before patches is DOA? I want to be super careful before merging here because it'll rebuild the docker image and potentially break many people's resources if wrong.

@mehtaphysical, please update the tests for your last commit. I've made a commitment on the docs for this resource to always have master passing.

**Comment by mehtaphysical** [comment]
@Karunamon yes. i'll do this in a bit

**Comment by aecepoglu** [comment]
@Karunamon To be honest I've already started using a patched version of this repo and doing extensive testing seems like a hassle to me.

**Comment by aecepoglu** [comment]
@Karunamon I've opened a newer PR ( pr #4 ). All checks passing and it returns the json value as it should.

**Comment by elementsweb** [comment]
Can we prioritise getting this fix into the master branch? Without the fix on line 66: `print('{ "version": { ref: "none" } }')` this resource simply does not work correctly (does update commit status okay though). When navigating to see information on a build (quite critical if you're a developer and want to see why something failed) you are just presented with 'loading' - after hours of debugging, the fix offered on line 66 resolves the issue.

**Pull request closed without merging**